docker/nfs-ganesha-ceph/nfs-ganesha-ceph-base/Dockerfile.j2

```diff
@@ -0,0 +1,30 @@
+FROM {{ namespace }}/{{ image_prefix }}openstack-base:{{ tag }}
```

Comment:
Because this is the only dockerfile you have, you can just name this `nfs-ganesha`. Also, per my other review, maybe this could be in the manila folder and even inherit from manila-base, if that makes sense?